Extensions/Guardrail+Settings.swift

```diff
@@ -116,7 +116,7 @@ public extension Guardrail where Value == HKQuantity {
             recommendedUpperBound = (recommendedHighScheduledBasalScaleFactor * highestScheduledBasalRate).matchingOrTruncatedValue(from: supportedBasalRates, withinDecimalPlaces: decimalPlaces)
             
             let absoluteBounds = highestScheduledBasalRate...absoluteUpperBound
-            let recommendedBounds = (recommendedLowerBound...recommendedUpperBound).clamped(to: absoluteBounds)
+            let recommendedBounds = recommendedLowerBound...recommendedUpperBound
```

Comment:
I'm not sure you can get rid of this particular clamping, particular the upper clamping to `absoluteUpperBound` which is calculated from lowest carb ratio.

Reply:
This clamping is also okay because the clamp, `absoluteBounds`, is already valid supported basal rates.

Reply:
Alternatively, you could have:
```
let recommendedBounds = recommendedLowerBound...min(recommendedUpperBound, absoluteUpperBound)
```

Reply:
Good catch!  Wow, you have eagle eyes.  And, in fact, this is an important thing to test.  I should add a unit test too.